db/schema.rb

```diff
@@ -111,6 +111,14 @@
     t.index ["updated_at"], name: "index_sessions_on_updated_at"
   end
 
+  create_table "subjects", force: :cascade do |t|
+    t.string "code", null: false
```

Comment:
Seems weird but the code and name are not required according to the API. Would be good to find out if we can make them required on the API or if there is a good reason for them to be allowed to be null

Reply:
i'll defer to API Sau @defong on this one! I guess modern languages being a 'bucket' course is one situation where the code needs to be null.

Reply:
The list https://github.com/DFE-Digital/teacher-training-api/blob/master/docs/api.md#list-of-known-subjects only really changes once a year, around Oct.

Let keep it strict/non-nullable until we know more of its usage.